### ISSUE TITLE
Accessibility violation where multiple tables in the same panel have the same search column

### ISSUE THREAD

**Issue by caritaou** [elyra-ai/canvas#2518]
ref: 4065

Since column names may not be easily changed to make it unique, we can use the table name in aria-labelledby
![Image](https://github.com/user-attachments/assets/37ebd9cf-d615-423e-bcb7-5a02c03ebc46)

**Comment by caritaou** [comment]
Unable to fix in application side, this will need to be addressed in Carbon https://github.com/carbon-design-system/carbon/issues/12360
